Block/Facebookopengraph.php

```diff
@@ -151,7 +155,10 @@ private function getPageTitle()
      */
     private function getCategoryDescription()
     {
-        return $this->getCategory()->getData('meta_description');
+        if($this->getCategory()->getData('meta_description')) {
```

Comment:
als eigene Variable wie mit Titel (vielleicht sogar in eigener Funktion die als Argument den Attribute namen verwendet)